2018/q4/product-documentation-writer.md

```diff
@@ -0,0 +1,66 @@
+# Product Documentation Writer
+
+## About npm
+
+Are you interested in joining the company that powers all web
+development?  npm is the package manager for JavaScript, the most
+important and fastest growing programming language on earth.  Over 11
+million developers use npm every day, at every company with a website.
+npm, Inc. is the company behind this revolution, developing products
+to bring value to enterprises in every vertical.
+
+## Our code of conduct
+
+npm is dedicated to providing a harassment-free experience for everyone, regardless of gender, sexual orientation, disability, age, physical appearance, body size, race, or religion. Sexual language and imagery is not appropriate in a professional environment. We do not tolerate harassment in any form and we are committed to partnering with you to foster a healthy work environment.
+
+## About this role
+
+We are looking for a product documentation writer to create useful, usable, and trustworthy help content for beginner to advanced npm users and customers. Your goal will be to empower our users to develop JavaScript applications with the npm CLI, website, and other products as easily and painlessly as possible.
+
+You will be responsible for planning, creating, and maintaining task-based documentation articles, API reference documentation, contextual help content, and other user assistance content as needed.
+
+Include 2 or 3 examples of your documentation work; they do not need to be related to npm.
+
+## Key responsibilities include
+
+- Creating a content plan for documenting new and updated features and products
+- Triaging and prioritizing content requests from across the company
+- Creating and updating documentation and other user assistance content
+- Gathering user and customer feedback from support requests and the npm community forum
+- Using metrics to plan documentation improvement projects
+
+## You will love this job if
+
+- You have experience using Jekyll or another static site generator for product documentation
+- You are comfortable using Git and GitHub in a documentation workflow
+- You have user experience design or UX research (or an interest in such)
```

Comment:
suggest "You have experience with, or interest in, user experience design or UX research"

Reply:
hm, I might remove this entirely -- I don't want to give the impression that this role is (definitely) a stepping stone into UX design or research.